wcivf/apps/people/management/commands/import_people.py

```diff
@@ -22,57 +25,97 @@ def add_arguments(self, parser):
         )
 
         parser.add_argument(
-            '--recent-minutes',
+            '--since',
             action='store',
-            dest='recent_minutes',
-            default=5,
-            type=int,
-            help='Number of minutes to look back for changes',
+            dest='since',
+            type=str,
```

Comment:
Do you think it is worth doing some kind of validation on this? For example, if I run `./manage.py import_people --since foo` it will just try and call https://candidates.democracyclub.org.uk/api/next/persons/?page_size=200&updated_gte=foo (which then throws a 500 in YNR).